### ISSUE TITLE
WARNING - GLIX MAY HAVE BEEN HACKED | PLEASE SEE

### ISSUE THREAD

**Issue by Zelkicks** [glixzzy/blooket-hack#1280]
Hackers will normally use bitcoin to get away with scamming people. It is untracable and unrefundable. DO NOT, and I repeat, DO NOT pay bitcoin for the cheats. Use GBasil until the hacks come back up. And if this issue gets taken down with no response from Glix, heed this warning

**Comment by coolman3289** [comment]
Agree, but how would a hacker activate the kill switch, unless it is a web hook.

**Comment by coolman3289** [comment]
Although, if I was a hacker, I would just edit it to include a token grabber for all your open tabs. No kid is going to pay 48,000 us dollars

**Comment by coolman3289** [comment]
Wait, do we have to pay it in all, or each person.

**Comment by Sanford0316** [comment]
I don't know its just stupid selling pretty much NFT's at this point but hacks

**Comment by Sanford0316** [comment]
but i do agree with gliz getting hacked
